### DISCUSSION TITLE
Site crashes if you go from pricing to homepage

### DISCUSSION THREAD

**Discussion by andredelotero** [planetscale/discussion#251] [Bug Reports] [answered]
If you open a new chrome tab (I'm using Chrome 104, Windows 10), type planetscale.com/pricing and then click on the logo to go to the homepage the site crashes:

![error](https://user-images.githubusercontent.com/80651774/184914000-8d4d56f5-9183-4eec-94d6-5eb125cd35c2.png)
![console](https://user-images.githubusercontent.com/80651774/184914009-290c4e1a-ad4c-4915-ad95-3ad0f82e3730.png)

**Comment by llibdude** [comment, marked as answer]
This has been fixed. Thank you for reporting!
